vergedb_test/routes/drones.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const router = express.Router();
+const drones = require('../services/drones');
+
+/* GET drones listing. */
+router.get('/', function (req, res, next) {
+    try {
+        //Commands here
+        res.json(drones.getMultiple(req.query.page));
+        
+    } catch (err) {
+        console.error(`Error while getting drones `, err.message);
+        next(err);
+    }
+});
+
+router.get('/:id', function (req, res) {
+    res.json(drones.getSingle(req.params.id));
```

Comment:
so if you requested `/drones/1234` and it doesn't exist, you'd get a `404` error code and a json response of something like:

```js
{ "error": "Drone 1234 not found" }
```